eng/tools/typespec-migration-validation/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@azure-tools/typespec-migration-validation",
+  "private": true,
+  "type": "module",
+  "main": "dist/src/index.js",
+  "bin": {
+    "tsmv": "cmd/tsmv.js"
+  },
+  "dependencies": {
+    "@azure-tools/typespec-autorest": ">=0.44.0 <1.0.0",
+    "fs": "^0.0.1-security",
+    "json-diff": "^1.0.6",
+    "yargs": "^17.7.2"
+  },
+  "devDependencies": {
+    "@types/jest": "^29.4.0",
+    "@types/json-diff": "^1.0.3",
+    "@types/node": "^18.19.86",
+    "@types/yargs": "^17.0.33",
+    "@typescript-eslint/eslint-plugin": "^5.54.0",
+    "@typescript-eslint/parser": "^5.54.0",
+    "eslint": "^8.35.0",
+    "jest": "^29.5.0",
```

Comment:
convert to use `vite` like the other folders under eng/tools

Reply:
I deleted jest. When we add tests I'll add vite.